source/cloud/cloud-integrations/cloud-bot-accounts.rst

```diff
@@ -93,9 +93,17 @@ What happens if a plugin is using a bot account that already exists as a user ac
 
 For a concrete example, suppose you enable the `Mattermost GitHub plugin <https://github.com/mattermost/mattermost-plugin-github>`_, which uses a ``github`` bot account, while an existing ``github`` user account was created for webhook integrations.
 
-Once the plugin is enabled, the plugin posts as the ``github`` account but without a `BOT` tag. An error message is logged to the server logs recommending the System Admin to convert the ``github`` user to a bot account by running ``mattermost user convert <username> --bot`` in the CLI.
+Once the plugin is enabled, the plugin posts as the ``github`` account but without a `BOT` tag. A System Administrator can convert the ``github`` user to a bot account by using ``mmctl``:
```

Comment:
Cloud users cannot access command line to execute mmctl commands. LInes 96 to 100 do not apply.